package.json

```diff
@@ -20,7 +20,7 @@
     "@accordproject/markdown-transform": "^0.16.19",
     "@accordproject/template-engine": "^2.3.5",
     "@monaco-editor/react": "^4.5.1",
-    "antd": "^5.7.2",
+    "antd": "^5.17.4",
```

Comment:
Can we remove this dependency now?

Reply:
Yeah, sure! Would make a refactor to all of these pretty soon including the conflicting files.